python/requirements_ml_docker.txt

```diff
@@ -1,7 +1,6 @@
 ipython
-# In TF >v2, GPU support is included in the base package.
-tensorflow==2.4.3
+tensorflow==2.5.0
 -f https://download.pytorch.org/whl/torch_stable.html
-torch==1.8.1+cu111
+torch==1.9.0+cu111
```

Comment:
maybe just get rid of this, and simply include cuda requirements in requirements.dl.txt as well?
gpu sounds pretty useful for DL as well.

Reply:
The `requirements_dl.txt` files is used for our CI (which currently is CPU only), so we have to make the distinction here.

Reply:
oh I see.
can you add a comment at the top of the file calling this out please.

Reply:
Added!